src/Workspaces/Core/Portable/EmbeddedLanguages/Json/IJsonNodeVisitor.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.EmbeddedLanguages.Json
```

Comment:
note: everything in this PR needs to move into teh Features project out of Workspaces (like we did for Regex).